src/com/company/SearchTheWord.java

```diff
@@ -0,0 +1,48 @@
+package com.company;
+
+import java.sql.SQLException;
+import java.util.StringTokenizer;
+
+import static com.company.ErrorMessage.SUCCESS_MESSAGE_WORD_FOUND;
+import static com.company.Main.*;
+
+public class SearchTheWord extends Thread {
+    String data;
+    int count;
+    String searchWord;
+    String txtFilePath;
+
+    public SearchTheWord(String data, int count, String searchWord, String txtFilePath) {
+        this.data = data;
+        this.count = count;
+        this.searchWord=searchWord;
```

Comment:
Formatting mistake, space missing before and after =
In line 19 as well

Reply:
file reformat has been done